moviepy/audio/fx/audio_volume_lr.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from moviepy.decorators import audio_video_fx
+
+
+@audio_video_fx
+def audio_volume_lr(clip, left=1, right=1):
+    """
+    For a stereo audioclip, this function enables to change the volume
+    of the left and right channel separately (with the factors `left`
+    and `right`). Makes a stereo audio clip in which the volume of left
+    and right is controllable.
+
+    Examples
+    ========
+
+    >>> music = AudioFileClip('music.ogg')
+    >>> audio_r = music.fx(audio_volume_lr, left=0, right=1)  # mute right channel
+    >>> audio_h = music.fx(audio_volume_lr, left=0.5, right=0.5)  # half audio
```

Comment:
```
    >>> audio_r = music.audio_volume_lr(left=0, right=1)  # mute right channel
    >>> audio_h = music.audio_volume_lr(left=0.5, right=0.5)  # half audio
```
I think that we'd prefer to push this method of using fx (particularly after #1340 which means that this will work no matter how you import `AudioFileClip`).